web_ui/src/pages/annotator/annotation/annotation-list/annotation-list.component.tsx

```diff
@@ -5,61 +5,23 @@ import { useMemo } from 'react';
 
 import { Content } from '@adobe/react-spectrum';
 import {
-    DragDropContext,
-    Draggable,
-    DraggableChildrenFn,
-    DraggableProvided,
-    Droppable,
-    DroppableProvided,
-    DropResult,
-} from 'react-beautiful-dnd';
-import { Virtuoso } from 'react-virtuoso';
+    ListBox as AriaComponentsListBox,
```

Comment:
Do we need to keep this renamed import?

Reply:
Yes, `@adobe/react-spectrum` also has a `ListBox` component, and I want to avoid confusion, especially since that one has different props and configuration options.